amplipi/updater/static/index.html

```diff
@@ -105,8 +98,133 @@ <h1><span class="text-white">Ampli</span><span class="text-danger">Pi</span></h1
+                  }
+                }
+
+                async function getPersist() {
+                  const response = await fetch("/settings/persist_logs", { method: "GET" });
+                  const data = await response.json();
+                  const checkbox = document.getElementById("persist-checkbox");
+                  const textbox = document.getElementById("persist-input");
+                  checkbox.checked = data.persist_logs;
+                  textbox.value = data.auto_off_delay;
+                }
+
+                async function setPersist() {
+                  const checkbox = document.getElementById("persist-checkbox");
+                  const textbox = document.getElementById("persist-input");
+
+                  const button = document.getElementById('persist-submit');
+                  const spinner = document.getElementById('persist-spinner');
+                  const success = document.getElementById('persist-success');
+                  const fail = document.getElementById('persist-fail');
+
+                  try {
+                    setVisibility(spinner, button, success, fail);
+                    const response = await fetch("/settings/persist_logs", {
+                      method: "POST",
+                      headers: {
+                          "Content-Type": "application/json",
+                      },
+                      body: JSON.stringify({
+                          persist_logs: checkbox.checked,
+                          auto_off_delay: parseInt(textbox.value),
+                      }),
+                    });
+                  if(response.ok){
+                    setVisibility(success, spinner, button, fail);
+                  } else {
+                    setVisibility(fail, success, spinner, button);
+                  }
+                  } catch { // The same as response.ok == false
+                    setVisibility(fail, success, spinner, button);
+                  }
+
+                  setTimeout(() => { // Reset to default after delay at the end
+                    setVisibility(button, spinner, success, fail);
+                  }, 1500);
+                }
+
+                window.onload = getPersist;
+              </script>
+
+              <div id="admin-settings-dialog">
+                This tab allows you to change settings relating to the core of AmpliPi
+              </div>
+
+              <!-- Password -->
+              <div id="set-password column-item">
+                <h6 class="hover-label">
+                  Set Password
+                  <span class="hover-tooltip">
+                    This form lets you set a web interface password, which will make it so that you must input the same password to access the system's controls. Leaving the fields empty will remove password protection. Note that this only affects the web interface, not the system password, and will log out any active sessions.
+                  </span>
```

Comment:
Both tooltips probably need some rewording

I moved them to be attached to the headers rather than to the input labels with the last commit to make them mobile compatible, previously you would've had to potentially inadvertently check the "enable" checkbox of persist logs to see what the tooltip has to say on mobile due to their lack of hoverover. This has had the effect of changing the context of the tooltips as well